ghost/core/core/server/services/members/members-api/controllers/RouterController.js

```diff
@@ -679,7 +680,15 @@ module.exports = class RouterController {
     }
 
     async _handleSignin(req, normalizedEmail, referrer = null) {
-        const {emailType} = req.body;
+        const {emailType, otc} = req.body;
+
+        const options = {
+            otc: false
+        };
+
+        if (labs.isSet('membersSigninOTC') && (otc === true || otc === 'true')) {
+            options.otc = true;
+        }
```

Comment:
Not keen on using `options` here as it creates some inconsistency with other methods, also change naming to `includeOTC` when value is a boolean rather than an actual OTC value.